docs/madr/decisions/057-rework-mesh-routes-matching.md

```diff
@@ -105,9 +105,15 @@ At the moment we can only target `Mesh*Routes` in MeshTimeout policy. This MADR
 Looking at what can be configured at Envoy route we can also apply this to our MeshRetry policy. Rate limit is also configurable on route,
 but we are configuring rate limit on inbound traffic, which does not apply to this MADR. 
 
+#### Producer/consumer model for targeting Mesh*Route in other policies
+
 It is worth to mention that rate limit would not work with our producer/consumer model. In our model consumer always has more priority
```

Comment:
I think we only support rate limit + routes if they're attached to a gateway, right? Otherwise rate limit is an inbound policy

Reply:
at the moment we can target whole `MeshGateway` not the route in MeshRateLimit, but since MeshGateway is cluster scoped, producer/consumer model does not apply to it

Reply:
Right I just mean I don't think it's relevant, MeshRateLimit targetting MeshHTTPRoute doesn't make sense independent of this MADR AFAICT